lib/StaticAnalyzer/Core/PlistDiagnostics.cpp

```diff
@@ -941,8 +943,10 @@ static MacroNameAndArgs getMacroNameAndArgs(SourceLocation ExpanLoc,
     return { MacroName, MI, {} };
 
   RawLexer.LexFromRawLexer(TheTok);
-  assert(TheTok.is(tok::l_paren) &&
-         "The token after the macro's identifier token should be '('!");
+  // When this is a functor-like token which is passed as parameter to another
+  // "function" token then its arguments are listed at another place.
```

Comment:
Could you please clarify a bit more? I never heard of the term "functor-like token" before. What does "another place" refer to? Do we really not know?